bindings/pydeck/pydeck/bindings/deck.py

```diff
@@ -188,3 +188,8 @@ def to_html(
             **kwargs,
         )
         return f
+
+    def _repr_html_(self):
```

Comment:
> Note that these are surrounded by single, not double underscores

I had to double check the docs but you're right. I guess double underscores are meant to be reserved for builtins only.